### PR TITLE
Finish up Priority

### LINKED ISSUE #61: Jiale - add priority
As an NUS computing student, I wish to add a priority level to each of my contacts, so that I can easily filter them by priority for future reference

### PR DESCRIPTION
**Pull request by Singa-pirate**
- Add priority in GUI
- Add assertion in `toString()` (resolves discussion in #82)
- More test cases for priority and related commands
- Fix errors in strings

Closes #61

### CODE REVIEW

File: src/test/java/networkbook/logic/commands/AddCommandTest.java [hunk 1]
```diff
@@ -61,6 +63,23 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
...
+
+        Person personInFilteredList = model.getFilteredPersonList()
+                .get(TypicalIndexes.INDEX_FIRST_PERSON.getZeroBased());
+
+        AddCommand addCommand = new AddCommand(TypicalIndexes.INDEX_FIRST_PERSON,
+                new EditPersonDescriptorBuilder().withPriority(CommandTestUtil.VALID_PRIORITY_AMY).build());
+
+        assertThrows(CommandException.class, () -> {
+            addCommand.execute(model);
+            addCommand.execute(model);
```
Comment (nknguyenhc) [inline src/test/java/networkbook/logic/commands/AddCommandTest.java]: Would need more explanation on what this test is doing here.

Comment (Singa-pirate) [reply src/test/java/networkbook/logic/commands/AddCommandTest.java]: It's to verify that CommandException is thrown when adding new priority to a person already with a priority.

Let me change the JUnit assertion to check the error message instead.

Comment (nknguyenhc) [reply src/test/java/networkbook/logic/commands/AddCommandTest.java]: Please change the name of the method to highlight that you are testing the priority only.
Other than that, everything else looks good! Let me know when you have done.

Comment (Singa-pirate) [reply src/test/java/networkbook/logic/commands/AddCommandTest.java]: Yep I'm done fixing the test case, please review again, thanks!

Comment (Singa-pirate) [reply src/test/java/networkbook/logic/commands/AddCommandTest.java]: Oh wait, I missed one line to fail the test case

Comment (Singa-pirate) [reply src/test/java/networkbook/logic/commands/AddCommandTest.java]: Okay done fixing it

File: src/test/java/networkbook/logic/commands/AddCommandTest.java [hunk 2]
```diff
@@ -61,6 +63,26 @@ public void execute_noFieldSpecifiedUnfilteredList_success() {
...
+                new EditPersonDescriptorBuilder().withPriority(CommandTestUtil.VALID_PRIORITY_AMY).build());
+
+        String expectedMessage = AddCommand.MESSAGE_MULTIPLE_UNIQUE_FIELD;
+
+        try {
+            addCommand.execute(model); // person would have priority
+            addCommand.execute(model); // add priority to the person again
+            fail();
+        } catch (CommandException e) {
+            assertEquals(expectedMessage, e.getMessage());
```
Comment (nknguyenhc) [inline src/test/java/networkbook/logic/commands/AddCommandTest.java]: Please refactor to make use of `networkbook.logic.commands.CommandTestUtil.assertCommandFailure` instead.

**Review by nknguyenhc — CHANGES_REQUESTED** [review]
Please make changes accordingly.

**Review by nknguyenhc — APPROVED** [review]
Looks very good!

**Pull request merged**